kolibri/plugins/document_epub_render/assets/src/views/EpubRendererIndex.vue

```diff
@@ -457,6 +447,32 @@
       delete global.ePub;
     },
     methods: {
+      updateProgress() {
+        if (this.locations.length > 0) {
+          this.$emit('updateProgress', this.sessionTimeSpent / this.expectedTimeToRead);
+        }
+      },
+      handleReadyRendition(width, height) {
+        this.updateRenditionTheme(this.themeStyle);
+
+        // this is not working, hence the delay via the debounce
+        // this.resizeRendition(width, height);
+        this.debounceResizeRendition(width, height);
+
+        this.rendition.on(EVENTS.RENDITION.RELOCATED, location => this.relocatedHandler(location));
+        this.rendition.on('keyup', this.handleKeyUps);
+        this.rendition.on('click', () => this.closeSideBar());
+
+        window.addEventListener('mousedown', this.handleMouseDown, true);
```

Comment:
Is this meant to be a passive listener? If so, should be `{ passive: true }` as the final arg.